File: chapter_02/MilesPerGallon.cpp
#include <iostream>
// Author: Daisy Hardwick
// Class: CIS-170
// Assignment: Miles-Per-Gallon C++ Program
using namespace std;

int main()
{
  // Initializing variables
  float milesDriven;
  float gasUsed;
  float gasMileage;

  // Getting the amount of miles driven
  cout << "How many miles have you driven?" << endl;
  cin >> milesDriven;

  // Getting the amount of gas used
  cout << "How many gallons of gas did you use?" << endl;
  cin >> gasUsed;

  // Preforming gas milage calculation
  gasMileage = milesDriven / gasUsed;

  // Displaying gas mileage
  cout << "You drove " << gasMileage << " miles per gallon of gas." << endl;
  return 0;
}

File: chapter_02/miles_per_gallon.py
# Author: Daisy Hardwick
# Class: CIS-170
# Assignment: Miles-Per-Gallon Python Program

# Gathering input for variables
miles_driven = int(input("How many miles have you driven?\n"))
gas_used = int(input("How many gallons of gas did you use?\n"))

#Performing gas mileage calculation
miles_per_gallon = miles_driven / gas_used

#Displaying gas mileage 
print(f"You drove {miles_per_gallon} miles per gallon of gas.")